### DISCUSSION TITLE
Minio Server authentication with LDAP and Local User

### DISCUSSION THREAD

**Discussion by suhailsid** [minio/minio#15899] [Q&A] [answered]
Hi team, any one know after minio new release support multiple IDP , is there any option I can use LDAP authentication with minio Local user which is already exist and working .  I am using latest Minio Release 

Thanks in Advance for your support.

**Comment by vadmeste** [comment, marked as answer]
No, this is not supported. Also I do not see this as a limitation, you can still create an LDAP user and create a new service account associated to it which will inherit the permissions of the LDAP user. You can even add more permission restriction to that.

**Reply by JgdKdoFhr** [reply]
It is a limitation though in moments when there is no ldap available. I always try to use LDAP authentication as main method and local users as a fallback.
